app/assets/stylesheets/application.scss

```diff
@@ -76,3 +75,17 @@
     margin: 0;
   }
 }
+
+#map-container {
+   height: 400px;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered border-color, border-radius, border-style, border-width, box-shadow, height, margin-top